app/install/dcos.md

```diff
@@ -75,8 +76,9 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
 
 5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.json` file from the
-    kong-dist-mesos repo, deploy Kong in the cluster:
+    Using the `kong_<postgres|cassandra>.json` file from the kong-dist-dcos
+    repo, deploy Kong in the cluster. Template registers Kong's `proxy` and
+    `admin`ports on Marathon-LB as service ports `10001` and `10002` respectively:
```

Comment:
Also, should `Marathon-LB` be formatted this way? Should it be `Marathan Loadbalancer`?